src/edu/washington/escience/myria/coordinator/catalog/MasterCatalog.java

```diff
@@ -1891,7 +1968,7 @@ protected Void job(final SQLiteConnection sqliteConnection) throws CatalogExcept
           try {
             SQLiteStatement statement =
                 sqliteConnection
-                    .prepare("UPDATE relations SET num_tuples=? WHERE user_name=? AND program_name=? AND relation_name=?;");
+                    .prepare("UPDATE relations SET num_tuples=? WHERE user_name=? AND program_name=? AND relation_name=? and tombstone=0;");
```

Comment:
Why would we want to silently fail here? Why not throw an exception on any update operations to tombstoned relation metadata?